hackathon_site/dashboard/frontend/src/slices/hardware/hardwareSlice.ts

```diff
@@ -126,6 +179,28 @@ const hardwareSlice = createSlice({
 
         builder.addCase(getHardwareWithFilters.rejected, (state, { payload }) => {
             state.isLoading = false;
+            state.isMoreLoading = false;
+            state.error = payload?.message || "Something went wrong";
+        });
+
+        builder.addCase(getHardwareNextPage.pending, (state) => {
+            state.isLoading = false;
```

Comment:
Since this thunk doesn't touch `isLoading` anymore, you don't actually have to explicitly set it in any of its reducers. Same with the `getHardwareWithFilters` reducers, you don't need to be setting `state.isMoreLoading` (unless I'm missing something?)

Reply:
yeah true, I was setting them just in case but I guess it's redundant